lock-api/src/main/java/com/palantir/lock/v2/LockImmutableTimestampResponse.java

```diff
@@ -21,7 +21,7 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
-// TODO(nziebart): also return a start timestamp here?
+// TODO(jkong): Can this be deprecated?
```

Comment:
Kind of strange to deprecate when it's used in StartAtlasDbTransactionResponse

Reply:
Fair enough - I think I should put this on the immutable timestamp _endpoint_, not this object!